### PR TITLE
ADEN-8791 Add option to force safe frame per slot

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/ad-engine/providers/gpt-provider.ts [hunk 1]
```diff
@@ -114,6 +114,7 @@ export class GptProvider implements Provider {
 		gptSlot.addService(window.googletag.pubads()).setCollapseEmptyDiv(true);
 
 		this.applyTargetingParams(gptSlot, targeting);
+		this.setSafeFrameStatus(gptSlot, adSlot);
```
Comment (fraszczakszymon) [inline src/ad-engine/providers/gpt-provider.ts]:
```suggestion
		if (adSlot.getConfigProperty('forceSafeFrame')) {
			this.setSafeFrameStatus(gptSlot);	
		}
```
What do you think?

File: src/ad-engine/providers/gpt-provider.ts [hunk 2]
```diff
@@ -142,6 +143,15 @@ export class GptProvider implements Provider {
 		Object.keys(targeting).forEach((key) => gptSlot.setTargeting(key, targeting[key]));
 	}
 
+	setSafeFrameStatus(gptSlot: googletag.Slot, adSlot: AdSlot) {
```
Comment (fraszczakszymon) [inline src/ad-engine/providers/gpt-provider.ts]: We can make this method `adSlot`-independent:
https://github.com/Wikia/ad-engine/pull/395/files#r285945976
What do you think?

File: src/ad-engine/providers/gpt-provider.ts (hunk 1 above)
Comment (dmnsobczak) [reply src/ad-engine/providers/gpt-provider.ts]: Why not, changed.

File: src/ad-engine/providers/gpt-provider.ts (hunk 2 above)
Comment (dmnsobczak) [reply src/ad-engine/providers/gpt-provider.ts]: Changed too.

**Pull request merged**